cloudbees-ci/configuration-as-code-examples/cloudbees-plugin-usage-plugin/configuration-as-code.yml

```
@@ -0,0 +1,3 @@
+tool:
+  cloudbeesPluginUsageAnalyzer:
+    enabled: true
```

This is the default configuration, so maybe we should change it to `enabled: false` instead?

```suggestion
    enabled: false
```

---

Yeah I think it makes more sense to display a fragment you would actually use.